browser/views.py

```diff
@@ -688,6 +689,39 @@ def delete_investigator_record(request, person_id, research_id):
 
 
 @staff_member_required
+def add_person(request):
+    if request.method == 'GET':
+        form = PersonForm()
+        extra_form = KnownPersonForm()
+
+    elif request.method == 'POST':
+        form = PersonForm(request.POST)
+        extra_form = KnownPersonForm(request.POST)
+        if form.is_valid():
+            Person.objects.create(
+                                 changed_by=request.user,
+                                 first_name=form.cleaned_data.get('first_name'),
+                                 last_name=form.cleaned_data.get('last_name')
+                             )
+            person = Person.objects.get(first_name=form.cleaned_data.get('first_name'),
```

Comment:
This has the same issue as checking if a person with that last and first name exists. It does not have to be the same person! There might be more than one with that name!!!

'create' should return the created object.

Reply:
Changed the code, the created object is returned instead of getting object by first name and last name.